crates/starknet-devnet-types/src/rpc/transactions/deploy_account_transaction_v1.rs

```diff
@@ -19,6 +19,7 @@ pub struct DeployAccountTransactionV1 {
     pub class_hash: ClassHash,
     pub contract_address_salt: ContractAddressSalt,
     pub constructor_calldata: Calldata,
+    #[serde(skip)]
     pub contract_address: ContractAddress,
```

Comment:
Isn't it a required parameter? Same for `crates/starknet-devnet-types/src/rpc/transactions/deploy_account_transaction_v3.rs`

Reply:
its not required for returning a transaction by hash. Only for transaction receipts